config.go

```diff
@@ -250,6 +252,17 @@ func ParseConfig(fn string) (Configuration, error) {
 		}
 	}
 
+	highlights_by_docs, ok := input.Get("bridge", "highlights_by_doc")
+	if ok {
+		b, err := strconv.ParseBool(highlights_by_docs)
+		if err == nil {
+			config.SupressHighlightsByDoc = b
+		}
+		config.SupressHighlightsByDoc = false
```

Comment:
FYI this looks like it would set `SupressHighlightsByDoc` to false no matter what